tls-attestation/diagrams/background-check-tls-handshake.txt

```diff
@@ -0,0 +1,64 @@
+.------------.       .-------.                .-------.      .----------.
+| Verifier X |       | TLS A |                | TLS B |      | Attester |
+'--+---------'       '---+---'                '---+---'      '----+-----'
+   |                     |                        |               |
+.--+-----------.         |                        |               |
+| TLS handshake |        |                        |               |
++--+------------+--------+------------------------+---------------+---.
+|  |  POST /newSession   |                        |               |    |
+|  |<--------------------+                        |               |    |
+|  | 201 Created         |                        |               |    |
+|  | Location: /76839A9E |                        |               |    |
+|  | Body: {             |                        |               |    |
+|  |   nonce,            |                        |               |    |
+|  |   supp-media-types  |                        |               |    |
+|  | }                   |                        |               |    |
+|  +-------------------->|                        |               |    |
+|  |                     | ClientHello            |               |    |
+|  |                     |  {...}                 |               |    |
+|  |                     |  attested-key-ext-req( |               |    |
+|  |                     |    nonce,              |               |    |
+|  |                     |    supp-media-types    |               |    |
+|  |                     |  )                     |               |    |
```

Comment:
Are the arrows between TLS A and TLS B missing here?

Reply:
Eagle eye! Fixed in [33d969b](https://github.com/hannestschofenig/tschofenig-ids/pull/53/commits/33d969b70480ce8966b6957d554def3f12998f9e)